helm-chart/csi-driver/values.yaml

```diff
@@ -11,6 +11,9 @@ enableMetrics: false
 # default metrics address port
 metricsPort: 8081
 
+# enable encryption for volumes by default
+volumeEncryption: false
+
```

Comment:
This comment is conflicting.

Question: Why are we setting this value at all? SInce its optional why not just keep it commented?